packages/block-editor/src/components/inserter/style.scss

```diff
@@ -639,3 +639,21 @@ $block-inserter-tabs-height: 44px;
 		height: 100%;
 	}
 }
+
+.block-editor-inserter__close.block-editor-inserter__close {
```

Comment:
Nothing to add to the other [ongoing conversation](https://github.com/WordPress/gutenberg/pull/47327/files?diff=unified&w=1#r1083330690) , but I was thinking that it may be good to add a little inline comment here to explain that these styles are meant to visually hide the button (while keeping accessible to assistive technology) unless the button itself is focused.